.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish Image on DockerHub
+
+# Action is triggered by pushing version tag to master
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+
+ build:
+    runs-on: ubuntu-latest
+    steps:
+    - 
```

Comment:
Did something format it for you this way? These new lines for each step after `-` seem surprising and obsolete. 
Why not simply `- uses: `, or `- name:` ?